allennlp/interpret/saliency_interpreters/integrated_gradient.py

```diff
@@ -30,9 +29,9 @@ def saliency_interpret_from_json(self, inputs: JsonDict) -> JsonDict:
             # Normalize results
             for key, grad in grads.items():
                 # The [0] here is undo-ing the batching that happens in get_gradients.
-                embedding_grad = numpy.sum(grad[0], axis=1)
```

Comment:
It's not obvious, but this line is also part of a dot product.  I don't remember why it's implemented this way, but it might be for consistency across the different interpreters, or there might be some efficiency considerations that I'm not remembering.  If you look at line 116 you'll see the first half of the dot product computation.